torch/optim/optimizer.py

```diff
@@ -650,6 +754,14 @@ def register_load_state_dict_pre_hook(
         calling ``load_state_dict`` on ``self``. The registered hook can be used to
         perform pre-processing before the ``load_state_dict`` call is made.
 
+        .. note::
+
+            There is an automatically registered load_state_dict pre-hook which
```

Comment:
Is it correct that because we always access the attribute before registering the user hook, the lazily added one will always be registered before any other hook?